posts/2022-07-20-liberty-tools-eclipse.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Introducing Liberty Tools for Eclispe"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/awisniew90
+author_github: https://github.com/awisniew90
+seo-title: Introducing Liberty Tools for Eclispe
+seo-description: A next generation plugin that brings the power of Open Liberty into your favorite IDE
+blog_description: "A next generation plugin that brings the power of Open Liberty into your favorite IDE"
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-actions.png
+---
+= Introducing Liberty Tools for Eclipse
+Adam Wisniewski <https://github.com/awisniew90>
+
+Liberty Tools for Eclipse provides a new experience for developing applications with Open Liberty. This early release offers Language Server integration, a Liberty Dashboard for organizing your projects, and "dev mode" from right within the IDE. 
```

Comment:
Liberty Tools for Eclipse => Liberty Tools (for Eclipse)

This early release provides support for editing Liberty configuration (server.xml) and for developing MicroProfile applications,a Liberty Dashboard

Reply:
I don't think "Language server integration" is something an end-user realizes they care about.. we need to externalize it a bit more.